### PR TITLE
Adds Javadocs and does minor refactoring

### PR DESCRIPTION
**Pull request by nxabdullah**

### CODE REVIEW

File: app/src/main/java/com/example/fitappa/View/MainActivity.java [hunk 1]
```diff
@@ -10,16 +10,20 @@
...
+     * @param savedInstanceState contains the data it was most recently supplied with by onSaveInstanceState
+     */
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
 
+
         // TODO: refactor and make this if statement be a method call in a gateway
+        // TODO: Turn this into a private method - Abdullah
```
Comment (1239uth) [inline app/src/main/java/com/example/fitappa/View/MainActivity.java:26]: Refactor into private method and remove return

File: app/src/main/java/com/example/fitappa/View/HomeActivity.java [hunk 2]
```diff
@@ -21,13 +25,10 @@ protected void onCreate(Bundle savedInstanceState) {
...
-            goBackToMain();
-        }
+        checkAuth();
 
         // Listeners
         openRoutinesBtn.setOnClickListener(v -> openRoutines());
-
         openProfileBtn.setOnClickListener(v -> openProfile());
 
         logoutBtn.setOnClickListener(v -> {
```
Comment (1239uth) [inline app/src/main/java/com/example/fitappa/View/HomeActivity.java:34]: Refactor to private method

**Review by 1239uth — CHANGES_REQUESTED** [review]
Do a code refactor on all files before committing. (Cmd + option + L)

File: app/src/main/java/com/example/fitappa/View/HomeActivity.java (hunk 2 above)
Comment (nxabdullah) [reply app/src/main/java/com/example/fitappa/View/HomeActivity.java:34]: Refactored

File: app/src/main/java/com/example/fitappa/View/MainActivity.java (hunk 1 above)
Comment (nxabdullah) [reply app/src/main/java/com/example/fitappa/View/MainActivity.java]: Refactored, please review again

**Comment by 1239uth** [comment]
Approved by me! Need 1 more reviewer, then we can merge.

File: .idea/modules/app/FitAppA.app.iml [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <facet type="android-gradle" name="Android-Gradle">
       <configuration>
         <option name="GRADLE_PROJECT_PATH" value=":app" />
-        <option name="LAST_SUCCESSFUL_SYNC_AGP_VERSION" value="4.2.2" />
+        <option name="LAST_SUCCESSFUL_SYNC_AGP_VERSION" value="4.2.0" />
```
Comment (sourenrex) [inline .idea/modules/app/FitAppA.app.iml:7]: I assume this won't cause an issue but is there a reason this was changed?

Comment (sourenrex) [reply .idea/modules/app/FitAppA.app.iml:7]: Other than that I think its good to merge

**Pull request merged**